### PR TITLE
add 2DC steps for platform

### PR DESCRIPTION
**Pull request by polarweasel**
Adding steps from Daniel to let Platform know about a 2DC deployment.

Also did some cleaning up on the 2DC page.

Deploy [preview is here](https://6050069484860700085e902b--infallible-bardeen-164bc9.netlify.app/latest/deploy/multi-dc/2dc-deployment/).

### CODE REVIEW

**Comment by polarweasel** [comment]
Adding @nmantrav2 and @vijiyb as I reworked the steps on the 2DC page, and i'd like an opinion on usability.

**Comment by daniel-yb** [comment]
adding @chirag-yb as a reviewer as per his request

File: docs/content/latest/deploy/multi-dc/2dc-deployment.md [hunk 1]
```diff
@@ -24,126 +23,128 @@ showAsideToc: true
...
+
+## Enable universes in Platform
+
+If you're using Yugabyte Platform to manage your universes, you need to call the following REST API endpoint on your Platform instance _for each universe_:
+
+```http
+PUT /api/customers/<myUUID>/universes/<myUniUUID>/setup_universe_2dc
+```
+
+Where `myUUID` is your customer UUID, and `myUniUUID` is the UUID of the universe (producer or consumer). The request should include an `X-AUTH-YW-API-TOKEN` header with your Platform API key.
```
Comment (daniel-yb) [inline docs/content/latest/deploy/multi-dc/2dc-deployment.md:141]: It would be nice to have some documentation eventually on how to get the API key in the platform UI and then link those docs here IMO. But this should be fine for now since we'll usually have customer support/ATS running customers through the setup process for now + they'll know how to generate the API keys.

**Review by daniel-yb — APPROVED** [review]
Overall looks good to me! Thanks for putting this together so quickly.

Maybe wait for someone like @chirag-yb to stamp too.

**Review by nmantrav2 — APPROVED** [review]

File: docs/content/latest/deploy/multi-dc/2dc-deployment.md (hunk 1 above)
Comment (chirag-yb) [reply docs/content/latest/deploy/multi-dc/2dc-deployment.md:141]: Lets add a screenshot for generating an API token and user permission needed for calling the API

Comment (chirag-yb) [reply docs/content/latest/deploy/multi-dc/2dc-deployment.md:141]: Need steps on how to find the customer and universes UUID

**Review by chirag-yb — CHANGES_REQUESTED** [review]
Added two comments for adding more information

File: docs/content/latest/deploy/multi-dc/2dc-deployment.md (hunk 1 above)
Comment (daniel-yb) [reply docs/content/latest/deploy/multi-dc/2dc-deployment.md:141]: @chirag-yb my worry about including those instructions in our docs now is that it might open a pandora's box on using YW APIs before we're ready for them to be publicly consumed. Thoughts?

File: docs/content/latest/deploy/multi-dc/2dc-deployment.md [hunk 2]
```diff
@@ -24,126 +23,168 @@ showAsideToc: true
...
-**YCQL example**
+Call `determine_repl_latency.sh -h` to get a summary of all command options.
+
+{{< /tip >}}
+
+**If you're using Yugabyte Platform to manage universes**, continue to the next section. If not, you're done!
+
+## Enable universes in Platform
+
+If you're using Yugabyte Platform to manage your universes, you need to call the following REST API endpoint on your Platform instance _for each universe_:
```
Comment (chirag-yb) [inline docs/content/latest/deploy/multi-dc/2dc-deployment.md:141]: @daniel-yb @polarweasel can you reword this to please? 

you need to call the following REST API endpoint on your Platform instance for each universe (producer and consumer) involved in 2DC replication

**Review by chirag-yb — APPROVED** [review]

**Comment by chirag-yb** [comment]
@polarweasel Looks good overall except one minor comment

**Review by VijiYB — APPROVED** [review]
Looks good
